tests/console/quota.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test that uses quota command line tool to test
+# This test consists on quota, configuring and use a regular user to test quota .
+# If succeed, the test passes, proving that the connection is working.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub setup {
```

Comment:
is it really needed to have it in function? so far setup is used only once

Reply:
done